src/index.html

```diff
@@ -1,17 +1,153 @@
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <label for="surname" class="formField">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label for="name" class="formField">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+        <label for="age" class="formField">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label for="date" class="formField">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            id="date"
+          >
+        </label>
+        <label for="accept" class="formField">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            id="accept"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <label for="email" class="formField">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label for="password" class="formField">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+        <label for="multi_choice" class="formField">
+          Do you love cats?
+          <input
+            type="radio"
+            name="yes"
+            id="yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="no"
+            id="no"
+          >
+          No
+        </label>
+        <label for="color" class="formField">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </label>
+        <label for="time" class="formField">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            id="time"
+          >
+        </label>
+        <label for="cars" class="formField">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
```

Comment:
it's better to pass lower-case values to save yourself from letter-case-driven issues
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```